app/controllers/ApplicationController.scala

```diff
@@ -387,4 +387,58 @@ trait ApplicationController extends BaseController with GatekeeperAuthWrapper {
     }
   }
 
+  def createPrivOrROPCApplicationPage(): Action[AnyContent] = { requiresRole(Role.APIGatekeeper, requiresSuperUser = true) {
+    implicit request =>
+      implicit hc => {
+        Future.successful(Ok(createPrivOrROPCApplication(createPrivOrROPCAppForm.fill(CreatePrivOrROPCAppForm(None, "", "", "")))))
```

Comment:
Could change case class to have default values and avoid the empty strings here.